kickstarts/base.ks.erb

```diff
@@ -157,6 +157,8 @@ open-vm-tools
 
 %post --log=/root/anaconda-post.log
 
+yum -y install http://www6.atomicorp.com/channels/atomic/centos/7/x86_64/RPMS/wmi-1.3.14-4.el7.art.x86_64.rpm
```

Comment:
I wasn't able to find it anywhere else either.  Where do we get the package we use for downstream builds?